src/test/java/hw2/TestBase.java

```diff
@@ -0,0 +1,49 @@
+package hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+
+import java.util.List;
+
+public  abstract class TestBase {
+    protected WebDriver driver;
+
+    @BeforeClass
+    public static void setupClass() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @BeforeMethod
+    public void setupTest() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+
+    }
+
+    protected WebElement waitAndGetElement(By by, int timeOutInSeconds) {
+        return new WebDriverWait(driver, timeOutInSeconds)
+                .until(ExpectedConditions.presenceOfElementLocated(by));
```

Comment:
This is a good catch to use **Explicit waits** in the test. Please read carefully the *Selenium Java Documentation* about methods which do you try to use. 
https://www.selenium.dev/selenium/docs/api/java/
```
public static ExpectedCondition<WebElement> presenceOfElementLocated(By locator)

An expectation for checking that an element is present on the DOM of a page. This does not necessarily mean that the element is visible.

Parameters:
    locator - used to find the element
Returns:
    the WebElement once it is located 
```

Please check all places where you use it